### PR TITLE
Added Romanian layout and translations with special romanian characters

### PR DESCRIPTION
**Pull request by bazyle**
* Added Romanian characters to the keyboard layout and changed the positioning corner for the secondary characters that were conflicting with the newly-introduced romanian characters
* Added Romanian translations

### CODE REVIEW

File: res/values-ro/strings.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" product="debug">Tastatură Unexpected (depanare)</string>
```
Comment (Julow) [inline res/values-ro/strings.xml:3]: The translation files must be formatted by running `python sync_translations.py`. (this helps maintaining them)

**Review by Julow — APPROVED** [review]
Many thanks! I can leave this open for a little more time to allow you to make additional changes.

Have you tested the layout on your phone ? You can find a debug build here: https://github.com/Julow/Unexpected-Keyboard/actions/runs/4878084051?pr=358

**Comment by bazyle** [comment]
> 

With very welcome! 😄 
Of course I tested it on my phone. I'm currently using the debug version on my phone as it works okay for me 👍

File: res/values-ro/strings.xml (hunk 1 above)
Comment (bazyle) [reply res/values-ro/strings.xml:3]: Thanks! I probably missed that. I'll push an update in a few moments.

**Review by Julow — COMMENTED** [review]
I've added more entries into `strings.xml` recently. Would you have time to translate them ?

You can directly do `git merge master` as I'll squash then `python sync_translations.py`.

**Comment by Julow** [comment]
I'll merge, thanks! I pushed some commits to your branch to fix the conflicts due to other changes.

**Pull request merged**